tests/integration/test_put_along_axis.py

```diff
@@ -132,18 +132,6 @@ def test_indices_bad_type(self, dtype):
         with pytest.raises(TypeError, match=msg):
             num.put_along_axis(self.a, ai, 100, axis=0)
 
-    @pytest.mark.xfail
```

Comment:
Is cuNumeric throwing the same type of exception as NumPy now? If so, then we'd want to keep this testcase, and remove the xfail.

We want to try and throw the same exception types as NumPy, at least when NumPy is doing some explicit error checking (we don't need to match the error message exactly).

Reply:
No, they are throwing different error messages. The implementation of put_along_axis is going through a little bit different path in cunumeric when compared to numpy. So it return error from a different API